sentry-android-core/src/main/java/io/sentry/android/core/AndroidLogger.java

```diff
@@ -9,7 +9,7 @@
 
   private static final String tag = "Sentry";
 
-  @SuppressWarnings("AnnotateFormatMethod")
+  @SuppressWarnings({"AnnotateFormatMethod", "LogConditional"})
```

Comment:
we check if logger is enabled by ourselves, we don't need to check the `BuildConfig.DEBUG`
Sentry logger != Android logger